pantheon-bundle/src/test/java/com/redhat/pantheon/html/HtmlTest.java

```diff
@@ -68,44 +68,6 @@ void encodeAllImageLocations() {
         });
     }
 
-    @Test
-    void dereferenceAllHyperlinks() {
```

Comment:
It definitely makes sense to remove this test since you're removing the code that it targets, but don't forget to add appropriate tests of your own for the new code that you're introducing!

Reply:
Yes Ben,
Once we have finalized and covered the cases in the code, I would write corresponding tests for them.